cluster/charts/rook-ceph/values.yaml

```diff
@@ -332,6 +332,9 @@ allowMultipleFilesystems: false
 # For more details see https://github.com/rook/rook/issues/2417
 enableSelinuxRelabeling: true
 
+# The duration between discovering devices in the rook-discover daemonset.
+discoverDevicesInterval: 60m
```

Comment:
I'm a bit conflicted about having the `discoverDevicesInterval` not in the `discover:` section above (line 320), but here.

What do you think?

Reply:
I don't really have a preference with that, I just wasn't sure if it belonged there. Want me to change it?

Reply:
I think it belongs in the `discover:` section.

Reply:
@BlaineEXE I have now added it to the discover: section. Thanks for the input